vent/coordinator/message.py

```diff
@@ -0,0 +1,89 @@
+from enum import Enum, auto
+
+
+class SensorValues:
+    def __init__(self, pip=None, peep=None, fio2=None, temp=None, humidity=None, vte=None, breaths_per_minute=None,
```

Comment:
I would love for this to be defined once somewhere in this repo. I have something similar in

 https://github.com/CohenLabPrinceton/Ventilator-Dev/blob/coordinator/vent/gui/defaults.py

but it would be great to unify this in one place -- i can adapt my code to read from it however you decide on doing it.

Reply:
How about let's first make this to be defined in both in GUI and coordinator? The advantage is that we don't need to maintain consistency. Later when the system is mature, we can merge them.